webpack.prod.js

```diff
@@ -23,6 +61,10 @@ module.exports = merge(common, {
     ]
   },
   plugins: [
+    new MiniCssExtractPlugin({
+      filename: 'style.[contentHash].css',
```

Comment:
Could you please explain why you added the content hash and ids below?

I don't fully understand and would like to!

Reply:
webpack provides a method of templating the filenames using bracketed strings called substitutions. The [contenthash] substitution will add a unique hash based on the content. so whenever the content is changed content hash will change which will result in the changed filename. and browser will load new file as name has been changed. if name is same browser loads cached files if names are same. so your deployed changes sometimes don't reflect to users. to overcome that issue I have added contentHash in output filename in prod.

Reply:
Oh so this is what helps with the cache busting. Got it, thanks!